src/Server.h

```diff
@@ -144,6 +152,9 @@ class cServer
 	from the settings. */
 	bool ShouldAllowMultiWorldTabCompletion(void) const { return m_ShouldAllowMultiWorldTabCompletion; }
 
+	/** Get the Forge mods registered for a given protocol. */
+	AStringMap & GetRegisteredForgeMods(const UInt32 a_Protocol);
```

Comment:
Make it return a `const AStringMap &`, so that the caller cannot modify the mod list after this call.